Apps/Src/Task_VoltTempMonitor.c

```diff
@@ -19,11 +20,24 @@ void Task_VoltTempMonitor(void *p_arg) {
     bool openWireHasBeenChecked = false;
     bool temperatureHasBeenChecked = false;
 
+    CANData_t CanData;
+    CanData.b = 0;
+    CanData.h = 0;
+    CanData.w = 0;
```

Comment:
In a union, all of the fields share the same memory, so you only have to set one of the fields

Reply:
In that case, I will only set the one that has data in it and I will remove all the 0's.